docs/source/getting-started/security-basics.rst

```diff
@@ -79,6 +79,9 @@ it is reasonable to run the hub without SSL.
 To achieve this, simply omit the configuration settings
 ``c.JupyterHub.ssl_key`` and ``c.JupyterHub.ssl_cert``
 (setting them to ``None`` does not have the same effect, and is an error).
+Reminder to ask: Does "simply omit" mean "make sure they are commented in 
+the configuration file" or "make sure they are set to an empty string" or
+something else?
```

Comment:
Omit means "not set", which is commented-out or removed from the file altogether (the file is Python, so comments are the same as not being there at all. This sentence is an attempt to state explicitly that setting them  to None or an empty value is not the same as not setting them.